ofl/kaiseitokumin/upstream.yaml

```diff
@@ -0,0 +1,8 @@
+branch: master
```

Comment:
upstream urls are also missing for these. Has something recently changed in the packager?

Reply:
I'm not sure. It looks like a lot of my PRs recently have had that field missing. I haven't been getting any warnings that the system is failing to locate that field or whathaveyou, so I'm pretty confused. Will investigate today. 

In the meantime, I'd suggest looking at the ones that are pure updates with dehinted fonts.

Reply:
urls have been re added so it is ready to be merged